systers_portal/meetup/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from meetup.views import MeetupLocationDetailView
+
+
+urlpatterns = [
+    url(r'^(?P<slug>\w+)/(?P<page_slug>\w+)/$',
```

Comment:
You are right. But the only other way that I can think of with just one template and only one slug in url(the first slug that will tell which meetup location) is to make our desired functionality possible by Javascript.

I cannot find/think any other Django way to achieve it. Do you have any suggestion?